.travis.yml

```diff
@@ -10,20 +10,20 @@ env:
   global:
   - DOCKER_REPO=$TRAVIS_REPO_SLUG
 before_install:
-- apt-cache madison docker-engine
-- travis_retry sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-engine
+- apt-cache madison docker-ce
```

Comment:
My sanity tests have gone well (it is a bit painful iterating quickly with our Vagrant setup with Storm 1.0+ and the junk it leaves in ZooKeeper for the blobstore).  So after we resolve whatever that issue is (be it a GitHub UI issue, or a bad merge of the changes) we can merge this.  I'll play around a bit with possibly cherry picking just your commits, maybe I'll have to do that for a clean merge.